packages/react/src/components/AppShell/app-shell.scss

```diff
@@ -33,16 +34,29 @@
     flex-grow: 1;
     transition: all 0.5s;
 
+    .oxygen-app-shell-navigation-wrapper {
+      background: var(--oxygen-palette-customComponents-Navbar-background);
+
+      .MuiDrawer-paper {
```

Comment:
Shall we add a custom class name to `.MuiDrawer-paper`? Something specific to Oxygen.

Reply:
Since this selects the `.MuiDrawer-paper` class under the `.oxygen-app-shell-navigation-wrapper` class, this can be considered oxygen specific, right?

Reply:
We can add this to the Navbar component's Drawer to make it oxygen specific `classes={{paper: 'oxygen-navbar-drawer-paper'}}`.